lemur/dns_providers/models.py

```diff
@@ -12,7 +12,7 @@ class DnsProvider(db.Model):
     __tablename__ = "dns_providers"
     id = Column(Integer(), primary_key=True)
     name = Column(String(length=256), unique=True, nullable=True)
-    description = Column(Text(), nullable=True)
+    description = Column(String(length=1024), nullable=True)
```

Comment:
This is change is done to match the definition of description column with actual schema definition from DB. This delta surfaced while creating a migration file. 
VARCHAR offers variable max size as compared to Text. This seems to be working well for Lemur, thus continuing with the same than doing a db change.